stage/testing/debian/bookworm/package-model.json

```diff
@@ -7,7 +7,7 @@
     "plymouth-theme-regolith": null,
     "regolith-avizo": null,
     "regolith-control-center": {
-      "ref": "r3_2-beta4-gnome-43",
+      "ref": "r3_3-alpha1-gnome-43",
```

Comment:
In the existing system, there is an implicit direct association between the staging directories and release level:

```
unstable = alpha
testing = beta
release = prod
```

Here it looks like the testing directory would model alpha (AKA 'unstable') versions.  Is my understanding correct? How does stage promotion work in terms of file mutations to the model directories post 3.2?

Reply:
Yes you are correct. This was unintentional. My idea was to get the point across that this is a very early testing tag (hence using alpha.)

Nothing will change in terms of tags and stage promotion post 3.2. They are just more automatic in terms of GitHub actions.